### ISSUE TITLE
autonum error: align environment

### ISSUE THREAD

**Issue by Konfekt** [MagicMuscleMan/autonum#2]
Under autonum 3.10, Texlive 2014, OpenSuse 13.2 there is an error message in all align environments
```
Misplaced alignment tab character &.
\math@cr@@@ ->&
               \omit \global \advance \row@ \@ne \ifst@rred \nonumber \fi \i...
l.51   \end{align}
```
The load order is fine and `hypertexnames=false`. Used to work fine before.

**Comment by MagicMuscleMan** [comment]
I cannot reproduce it here (autonum 3.1, Texlive 2015dev, Debian testing). Can you please provide a minimal example?
Did you really test autonum 3.9? I ask because the difference between 3.9 and 3.10 is really small and I rather assume, that the problem came up with 3.8.

**Comment by Konfekt** [comment]
Hmm.. the last running version was that provided by Suse from 2013-04-28 21:51:31. 

I'll try to provide a minimal example. It is under scrartcl, hyperref and cleverref.

**Comment by MagicMuscleMan** [comment]
I got another bug report which showed the problem. The problem only occured, if there was no token between a newline (\) and and alignment tab character (&). This is now fixed in Master and will be part of 0.3.11.

**Comment by Konfekt** [comment]
Thanks! (Perhaps drop a line when it hit the texlive repo.)
